.gitignore

```diff
@@ -5,6 +5,9 @@
 *.d
 !*.d/
 *.dSYM
+*.ddc
```

Comment:
why add? please drop the change in .gitignore

Reply:
Hmm, I'm not sure where this change came from - it must have slipped in during a merge / rebase.  I'll remove all changes to .gitignore.